go/vt/vtctl/reparentutil/replication_test.go

```diff
@@ -161,27 +161,41 @@ func TestFindValidEmergencyReparentCandidates(t *testing.T) {
 			shouldErr: false,
 		},
 		{
-			name: "tablet with errant GTIDs is excluded",
+			name: "tablet with superset GTIDs is included",
```

Comment:
Nit, but I think the intention would be more clear with something like:
```suggestion
			name: "exactly 2 tablets, 1 with GTID superset, neither considered errant",
```

If it's clear to you though then no need.

Reply:
We can rewrite these tests altogether when we review FindErrantGTIDs with a larger group. I'm not satisfied with them.